chstrings/__init__.py

```diff
@@ -21,6 +21,12 @@ def _preprocess_variables(config, strings):
     else:
         strings['lead_section_hint'] = ''
 
+    reliable_sources_hint_link = _link(
+        config.reliable_sources_link,
+        config.reliable_sources_link_title)
+    strings['reliable_sources_hint'] = \
+        flask.Markup(strings['reliable_sources_hint']) % reliable_sources_hint_link
```

Comment:
I don't see a `reliable_sources_hint` string, maybe it got lost while you shuffled commits around?

This should be `instructions_goal`, and handle a few different cases:

1. We might have an outdated string that doesn't allow interpolating the link at all
2. We might have an up-to-date string, but no URL in the config
3. We might have both the link and up-to-date string